client/components/common/NavBar.js

```diff
@@ -6,11 +6,17 @@ import PropTypes from 'prop-types';
 import SearchBar from './SearchBar';
 import NotificationCenter from './NotificationCenter';
 import UserProfile from './UserProfile';
+import SideNav from './SideNav';
 import LogoReplacement from '../../assets/icons/LogoReplacement';
 
 // assets
 import '../../assets/components/navbar.scss';
```

Comment:
Also I dont think we need this since import it here `https://github.com/AndelaOSP/Andela-Socials/pull/144/files#diff-c0c3b647bdf860f920d5e6da584fef1dR10`

Reply:
The sidenav needs to be imported here since it is in a separate file. What we have on line 12 is a style import. Not sure why it was imported into the component directly though.